### ISSUE TITLE
Requiring fs-extra makes binary nonexecutable for child_process

### ISSUE THREAD

**Issue by sergiz** [vercel/pkg#850]
If I require fs-extra package in my entry script, then using fork or spawn on another script from the package, it errors out with EACCES (it appears as the binary becomes nonexecutable). However, if you re-run the binary, sometimes it will start working on second attempt.

Anyone else experiencing the same problem?

The problem seems to appear when the script that spawns child process requires fs-extra, and the "child process" script does it too. If either of them do not require fs-extra, then everytihng works fine. I tested with node10 and node12.

Edit: "request" is also one of the packages that produces the same problem.

**Comment by sergiz** [comment]
Not stale.

**Comment by robertsLando** [comment]
@jesec @leerob Any clue? I know that we actually patch the fs module in nodejs source to make pkg work, afaik fs-extra should extend it but maybe could cause some conflicts?

**Comment by fcastilloec** [comment]
`fs-extra` released v10 and now it's broken with `pkg`. Here's the error I'm seeing:
```
TypeError: Cannot read property 'name' of undefined
    at /snapshot/project/node_modules/universalify/index.js:15:26
    at Object.<anonymous> (/snapshot/project/node_modules/fs-extra/lib/fs/index.js:57:27)
    at Module._compile (pkg/prelude/bootstrap.js:1503:22)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at Module.require (pkg/prelude/bootstrap.js:1409:31)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/snapshot/project/node_modules/fs-extra/lib/index.js:5:6)
```

Using `fs-extra@9` doesn't have this problem.

**Comment by robertsLando** [comment]
@fcastilloec this has been fixed in #1173 wait for next release :)
